<launch>
	<node pkg="stage_ros" type="stageros" name="stageros" args="$(find learning_ros_sandbox)/worlds/map1_box2.world" respawn="false" output="screen" />

	<node pkg="learning_ros_sandbox" type="box3.py" name="sandbox_3"/>

	<param name="point_A" type="str" value="1,1" />
	<param name="point_B" type="str" value="5,1" />

</launch>